### ISSUE TITLE
contrast of disabled form input

### ISSUE THREAD

**Issue by goetsu** [18F/web-design-standards#757]
on https://playbook.cio.gov/designstandards/form-controls/ the accessibility guidelines state : 
"If you adjust the color scheme of the buttons, ensure a minimum contrast ratio of 4.5:1 (for small text, 3:1 for large) for all states of the button. This includes default, hover, selected, and disabled."

WCAG 2.0 doesn't require minimum contrast ratio for disabled interface component because otherwise user can't know it's disabled cf : WCAG 2.0 1.4.3 criteria "Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement."

**Comment by maya** [comment]
Thanks for the input, you make a good point. Are you saying to remove disabled from our list in our accessibility guidelines as items that must meet AA color contrast?

And change to something like this?

>If you adjust the color scheme of the buttons, ensure a minimum contrast ratio of 4.5:1 (for small text, 3:1 for large) for the default, hover, and selected states of the button. The disabled state may have less contrast to indicate that it is inactive.

**Comment by goetsu** [comment]
yes that it but to be more accurate maybe you could include focus state : 
"If you adjust the color scheme of the buttons, ensure a minimum contrast ratio of 4.5:1 (for small text, 3:1 for large) for the default, hover, focus and selected states of the button. The disabled state may have less contrast to indicate that it is inactive."
